src/components/Terminal/_terminal.scss

```diff
@@ -36,7 +40,6 @@ $terminal-header-height: 40px;
 
     .is-visible & {
       transform: rotate(0);
```

Comment:
Doesn't this need a animation?

Reply:
There's another issue to remove this anyway.